egs2/TEMPLATE/svs1/README.md

```diff
@@ -0,0 +1,540 @@
+```yaml
+# e.g.
+svs_conf:
+    spks: 5  # Number of speakers
+```
+Please run the training from stage 6.
+```sh
+$ ./run.sh --stage 6 --use_sid true --train_config /path/to/your_multi_spk_config.yaml
+```
+
+### Multi-language model with language ID embedding training
+
+First, you need to run from the stage 2 and 3 with `--use_lid true` to extract speaker ID.
+```sh
+$ ./run.sh --stage 2 --stop-stage 3 --use_lid true
+```
+You can find the speaker ID file in `dump/raw/*/utt2lid`.
+**Note that you need to additionally create `utt2lang` file in stage 1 to generate `utt2lid`.**
+Then, you can run the training with the config which has `langs: #langs` in `svs_conf`.
+```yaml
+# e.g.
+svs_conf:
+    langs: 4  # Number of languages
+```
+Please run the training from stage 6.
+```sh
+$ ./run.sh --stage 6 --use_lid true --train_config /path/to/your_multi_lang_config.yaml
+```
+
+Of course you can further combine with speaker ID embedding.
+If you want to use both sid and lid, the process should be like this:
+```sh
+$ ./run.sh --stage 2 --stop-stage 3 --use_lid true --use_sid true
+```
+Make your config.
+```yaml
+# e.g.
+svs_conf:
+    langs: 4   # Number of languages
+    spks: 5    # Number of speakers
+```
+Please run the training from stage 6.
+```sh
+$ ./run.sh --stage 6 --use_lid true --use_sid true --train_config /path/to/your_multi_spk_multi_lang_config.yaml
+```
+
+
+### Vocoder training
+
+If your `--vocoder_file` is set to none, Griffin-Lim will be used.
+You can also train corresponding vocoder using [kan-bayashi/ParallelWaveGAN](https://github.com/kan-bayashi/ParallelWaveGAN)..
+
+Pretrained vocoder is like follows:
+
+```sh
+*_hifigan.v1 
+├── checkpoint-xxxxxxsteps.pkl
+├── config.yml
+└── stats.h5
+```
+
+```sh
+# Use the vocoder trained by `parallel_wavegan` repo manually
+$ ./run.sh --stage 7 --vocoder_file /path/to/checkpoint-xxxxxxsteps.pkl --inference_tag decode_with_my_vocoder
+```
+
+### Evaluation
+
+We provide four objective evaluation metrics:
+
+- Mel-cepstral distortion (MCD)
+- Logarithmic rooted mean square error of the fundamental frequency (F![1](http://latex.codecogs.com/svg.latex?_0)RMSE)
+- Semitone accuracy (Semitone ACC)
+- Voiced / unvoiced error rate (VUV_E)
+
+For MCD, we apply dynamic time-warping (DTW) to match the length difference between ground-truth singing and generated singing.
+
+Here we show the example command to calculate objective metrics:
+
+```sh
+cd egs/<recipe_name>/svs1
+. ./path.sh
+# Evaluate MCD
+./pyscripts/utils/evaluate_mcd.py \
+    exp/<model_dir_name>/<decode_dir_name>/eval/wav/gen_wavdir_or_wavscp.scp \
+    dump/raw/eval/gt_wavdir_or_wavscp.scp
+```
+```sh
+cd egs/<recipe_name>/svs1
+. ./path.sh
+# Evaluate log-F0 RMSE & Semitone ACC & VUV Error Rate
+./pyscripts/utils/evaluate_f0.py \
+    exp/<model_dir_name>/<decode_dir_name>/eval/wav/gen_wavdir_or_wavscp.scp \
+    dump/raw/eval/gen_wavdir_or_wavscp.scp
+```
```

Comment:
We've changed it into three separate files for log-F0 RMSE & Semitone ACC & VUV Error Rate.